app/src/components/market/sections/market_profile/market_status/closed.tsx

```diff
@@ -475,28 +470,25 @@ const Wrapper = (props: Props) => {
                 </StyledButtonContainer>
               ) : (
                 <>
-                  {!isConditionResolved && (
-                    <BorderedButtonContainer>
-                      <Button
-                        buttonType={ButtonType.primary}
-                        disabled={status === Status.Loading}
-                        onClick={resolveCondition}
-                      >
-                        Resolve Condition
-                      </Button>
-                    </BorderedButtonContainer>
-                  )}
-                  {isConditionResolved && hasWinningOutcomes && (
-                    <BorderedButtonContainer>
-                      <Button
-                        buttonType={ButtonType.primary}
-                        disabled={status === Status.Loading}
-                        onClick={() => redeem()}
-                      >
-                        Redeem
-                      </Button>
-                    </BorderedButtonContainer>
-                  )}
+                  <BorderedButtonContainer
+                    borderTop={
+                      (!isConditionResolved && hasWinningOutcomes) || (isConditionResolved && hasWinningOutcomes)
+                        ? true
+                        : false
```

Comment:
We are almost getting there! So why do you need ternary operators here? Since if it evaluates to true it will return true if not it will return  false...So i would just omit them....heres link to learn more  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator .

Reply:
👍